app/views/split_checkout/_payment.html.haml

```diff
@@ -1,7 +1,7 @@
 .medium-6
   - if @order.distributor.vouchers.present?
     %div.checkout-substep
-      = render partial: "split_checkout/voucher_section", locals: { order: @order, voucher_adjustment: @voucher_adjustment }
+      = render partial: "split_checkout/voucher_section", locals: { order: @order, voucher_adjustment: @order.voucher_adjustments.first }
```

Comment:
Same as my previous comment, I did it that way so the view "isn't the one doing the query". 
Now that I think about it, the view would still be doing the query as I assume the query will happen when the view try to use `@voucher_adjustment`. :shrug: